### PR TITLE
Made Scav Commanders recognisable as real commanders in some gadgets and widgets

### PR DESCRIPTION
**Pull request by Damgam**
Made Scav Commanders recognisable as real commanders in some gadgets and widgets

### CODE REVIEW

File: language/en/units.json [hunk 1]
```diff
@@ -8,7 +8,10 @@
 		},
 		"dead": "%{name} Wreckage",
 		"heap": "%{name} Debris",
+		"decoycommander": "Decoy",
 		"scavenger": "Scavenger %{name}",
+		"scavcommandernametag": "Scav Commander",
+		"scavdecoycommandernametag": "Scav Decoy",
```
Comment (WatchTheFort) [inline language/en/units.json:14]: Scavengers should always be referred to as Scavengers on the UI, never as Scavs

File: luaui/Widgets/gui_com_nametags.lua [hunk 2]
```diff
@@ -144,23 +144,38 @@ local function GetCommAttributes(unitID, unitDefID)
 	local playerRank
 	local name = ''
 	local luaAI = Spring.GetTeamLuaAI(team)
+	Spring.Echo("IsDecoy:", UnitDefs[unitDefID].customParams.decoyfor)
 	if luaAI and luaAI ~= "" and string.find(luaAI, 'Scavengers')  then
-
+		--name = "Scav Commander" -- todo: i18n this thing
+		if UnitDefs[unitDefID].customParams.decoyfor then
+			name = Spring.I18N('units.scavdecoycommandernametag', {})
```
Comment (WatchTheFort) [inline luaui/Widgets/gui_com_nametags.lua:151]: I18N without any interpolated data don't need to be passed an empty table.
`name = Spring.I18N('units.scavdecoycommandernametag')

File: language/en/units.json (hunk 1 above)
Comment (WatchTheFort) [inline language/en/units.json:14]: Translation keys should be in camelCase

Comment (Damgam) [reply language/en/units.json:14]: > Scavengers should always be referred to as Scavengers on the UI, never as Scavs

Scav units are all named as "Scav Name" on the UI so this is consistent with that.

File: luaui/Widgets/gui_com_nametags.lua (hunk 2 above)
Comment (Damgam) [reply luaui/Widgets/gui_com_nametags.lua:151]: > I18N without any interpolated data don't need to be passed an empty table. `name = Spring.I18N('units.scavdecoycommandernametag')`

Leftover from me trying to get this thing to work, will fix if I don't forget.

File: language/en/units.json (hunk 1 above)
Comment (Damgam) [reply language/en/units.json:14]: sure, will fix.

Comment (WatchTheFort) [reply language/en/units.json:14]: That has to change

Comment (sprunk) [reply language/en/units.json:14]: why? scav is a far better name

Comment (WatchTheFort) [reply language/en/units.json:14]: The game UI should never present any slang, jargon, acronyms, or abbreviations to the player

Comment (sprunk) [reply language/en/units.json:14]: rename scavs to scavs then

**Pull request merged**